fields/types/password/test/type.js

```diff
@@ -8,6 +8,376 @@ exports.initList = function (List) {
 };
 
 exports.testFieldType = function (List) {
+	describe('updateItem', function () {
+		it('should update password if specified', function (done) {
+			var testItem = new List.model();
+			List.fields.password.updateItem(testItem, {
+				password: 'asdf',
+			}, function () {
+				demand(testItem.password).be('asdf');
+				done();
+			});
+		});
+
+		it('should update password with hash if specified', function (done) {
+			var testItem = new List.model();
+			List.fields.password.updateItem(testItem, {
+				password_hash: '12asdf34',
+			}, function () {
+				demand(testItem.password).be('12asdf34');
+				done();
+			});
+		});
+
+		it('should update password if both password and hash specified', function (done) {
+			var testItem = new List.model();
+			List.fields.password.updateItem(testItem, {
+				password: 'asdf',
+				password_hash: '12asdf34',
+			}, function () {
+				demand(testItem.password).be('asdf');
+				done();
+			});
+		});
+
+		it('should clear password if passed password is null', function (done) {
+			var testItem = new List.model({
+				password: 'asdf',
+			});
+			List.fields.password.updateItem(testItem, {
+				password: null,
+			}, function () {
+				demand(testItem.password).be.null();
+				done();
+			});
+		});
```

Comment:
Yep, that should clear the stored value. We should actually query and assert the stored value after a save to ensure we're not saving a hash of `null` (if that's even a thing)